Airports/EGCM/SMR/Geo.txt

```diff
@@ -0,0 +1,76 @@
+N053.50.36.823 W001.12.13.882 N053.50.36.334 W001.12.13.785 smrWhite
+N053.50.36.334 W001.12.13.785 N053.50.31.344 W001.12.10.007 smrWhite
+N053.50.31.344 W001.12.10.007 N053.50.31.998 W001.12.09.663 smrWhite
+N053.50.31.998 W001.12.09.663 N053.50.32.475 W001.12.09.438 smrWhite
+N053.50.32.475 W001.12.09.438 N053.50.32.964 W001.12.08.798 smrWhite
+N053.50.32.964 W001.12.08.798 N053.50.33.338 W001.12.07.990 smrWhite
+N053.50.33.338 W001.12.07.990 N053.50.33.495 W001.12.07.583 smrWhite
+N053.50.33.495 W001.12.07.583 N053.50.31.787 W001.12.06.205 smrWhite
+N053.50.31.787 W001.12.06.205 N053.50.32.196 W001.12.06.879 smrWhite
+N053.50.32.196 W001.12.06.879 N053.50.32.335 W001.12.07.487 smrWhite
+N053.50.32.335 W001.12.07.487 N053.50.32.282 W001.12.08.247 smrWhite
+N053.50.32.282 W001.12.08.247 N053.50.31.962 W001.12.08.804 smrWhite
+N053.50.31.962 W001.12.08.804 N053.50.31.632 W001.12.09.214 smrWhite
+N053.50.31.632 W001.12.09.214 N053.50.31.018 W001.12.09.271 smrWhite
+N053.50.31.018 W001.12.09.271 N053.50.30.326 W001.12.09.130 smrWhite
+N053.50.30.326 W001.12.09.130 N053.50.29.736 W001.12.08.682 smrWhite
+N053.50.29.736 W001.12.08.682 N053.50.28.031 W001.12.07.372 smrWhite
+N053.50.28.031 W001.12.07.372 N053.50.26.263 W001.12.06.057 smrWhite
+N053.50.26.263 W001.12.06.057 N053.50.23.493 W001.12.04.027 smrWhite
+N053.50.23.493 W001.12.04.027 N053.50.20.395 W001.12.01.147 smrWhite
+N053.50.20.395 W001.12.01.147 N053.50.17.810 W001.11.58.636 smrWhite
+N053.50.17.810 W001.11.58.636 N053.50.16.663 W001.11.57.628 smrWhite
+N053.50.16.663 W001.11.57.628 N053.50.15.841 W001.11.57.025 smrWhite
+N053.50.15.841 W001.11.57.025 N053.50.13.743 W001.11.55.481 smrWhite
+N053.50.13.743 W001.11.55.481 N053.50.12.316 W001.11.54.522 smrWhite
+
+; - Tower_apron
+N053.50.06.888 W001.11.56.784 N053.50.07.568 W001.11.58.260 smrWhite
+N053.50.07.568 W001.11.58.260 N053.50.07.517 W001.11.58.733 smrWhite
+N053.50.07.517 W001.11.58.733 N053.50.07.800 W001.11.59.088 smrWhite
+N053.50.07.800 W001.11.59.088 N053.50.08.436 W001.11.58.234 smrWhite
+N053.50.08.436 W001.11.58.234 N053.50.08.288 W001.11.57.698 smrWhite
+N053.50.08.288 W001.11.57.698 N053.50.07.711 W001.11.55.946 smrWhite
+N053.50.07.711 W001.11.55.946 N053.50.07.738 W001.11.55.830 smrWhite
+
+;Holds
+; - F1
+N053.50.01.607 W001.11.59.518 N053.50.01.884 W001.11.58.804 smrWhite
+; - A1
+N053.49.52.516 W001.12.24.144 N053.49.52.479 W001.12.23.315 smrWhite
+; - A2
+N053.50.09.551 W001.11.54.493 N053.50.09.401 W001.11.53.695 smrWhite
+; - A3
+N053.50.12.116 W001.11.53.207 N053.50.11.817 W001.11.52.481 smrWhite
+; - B3
+N053.50.14.794 W001.11.46.302 N053.50.14.291 W001.11.46.015 smrWhite
+; - B2
+N053.50.15.670 W001.11.42.198 N053.50.15.258 W001.11.41.914 smrWhite
+; - B1
+N053.50.23.656 W001.11.05.825 N053.50.23.954 W001.11.05.057 smrWhite
+; - E2
+N053.50.13.815 W001.11.42.353 N053.50.14.257 W001.11.41.958 smrWhite
+; - E1
+N053.50.11.188 W001.11.37.752 N053.50.11.454 W001.11.37.166 smrWhite
```

Comment:
Single lines - comments at end of line (the ` - ` is unnecessary), and ensure alphabetical order.